sql/error_handler.h

```diff
@@ -219,7 +220,7 @@ class No_such_table_error_handler : public Internal_error_handler {
   virtual bool handle_condition(THD *, uint sql_errno, const char *,
                                 Sql_condition::enum_severity_level *,
                                 const char *) {
-    if (sql_errno == ER_NO_SUCH_TABLE) {
+    if (sql_errno == ER_NO_SUCH_TABLE || sql_errno == ER_BAD_DB_ERROR) {
```

Comment:
This won't silence the no database exists error in any other case?

Reply:
In my opinion it will not be a problem:
1. No MTR/unit test failures because of this
2. No_such_table_error_handler used in 3 places. In all of them it is used to suppress the error caused by nonexistent table (table does not exist at all or removed in parallel session). Table could be removed together with database